2.Lets-Build-OTPublisher/Lets-Build-OTPublisher/ViewController.m

```diff
@@ -74,7 +74,7 @@ - (BOOL)shouldAutorotateToInterfaceOrientation:
  */
 - (void)doConnect
 {
-    OTError *error;
+    OTError *error = nil;
```

Comment:
ARC does this for you, which we don't use in a lot of places internally. Not sure about the samples though. http://stackoverflow.com/a/9072040/1308729